server/src/main/java/io/deephaven/server/barrage/BarrageStreamGenerator.java

```diff
@@ -558,6 +649,56 @@ private ByteBuffer getMetadata(final SubView view) throws IOException {
         return header.dataBuffer().slice();
     }
 
+    private ByteBuffer getSnapshotMetadata(final SnapshotView view) throws IOException {
```

Comment:
If we *do* get rid of this (or narrow it), we'll need changes to `BarrageTable` to supply empty values or flat row sets for the essential fields.

Reply:
This needs to be deleted, per Larry.